packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -541,5 +541,7 @@ export const messages = {
     'Package ID should be a 15 or 18 character Id that starts with 04t',
   package_id_gatherer_placeholder: '04t...',
   force_function_enter_function: 'Enter a name for the function',
-  force_function_enter_language: 'Select a language for your function'
+  force_function_enter_language: 'Select a language for your function',
```

Comment:
@jag-sfdc - when this pops up will we provide a list of valid languages?

Reply:
Yes, this shows a list of languages - javascript and typescript.